pages/frontend/work.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Work</title>
+        <link rel="stylesheet" type="text/css" href="../../css/app.css" />
+    </head>
+    <body>
+        <nav class="navbar-container">
+            <div class="navbar-logo">
+                <p>Business</p>
+            </div>
+            <div class="navbar-items">
+                <ul class="nav-item-list">
+                    <li><a href="../../index.html" >About</a></li>
+                    <li><a href="work.html" class="active">work</a></li>
+                    <li><a href="team.html">team</a></li>
+                    <li><a href="#">blog</a></li>
+                    <li><a href="#">contact</a></li>
+                </ul>
+            </div>
+            <div class="navbar-actions">
+                <button class="primary-button">Contact Us</button>
+            </div>
+        </nav>
+
+        <div class="banner-container">
+            <p class="work-banner-text">Our Work</p>
+        </div>
+
+        <div class="team-projects">
+            <div class="services-header">
+                <p>Our Work</p>
+                <p> Team projects</p>
+            </div>
+        </div>
```

Comment:
WHy do we need to wrap this to another div `team-projects`? 
```suggestion
        <div class="services-header">
            <p>Our Work</p>
            <p> Team projects</p>
        </div>
```